.docker/php-fpm.conf

```
@@ -0,0 +1,6 @@
+[global]
+
+[www]
+php_flag[display_errors] = on
+php_admin_value[error_log] = /var/log/fpm-php.www.log
+php_admin_flag[log_errors] = on
```

```suggestion
php_admin_flag[log_errors] = on
php_admin_flag[post_max_size] = 100M
php_admin_flag[upload_max_filesize] = 100M
```

`SHOULD` - Now that this is in a separate file, the `if you'd like too...` seems like an artifact leftover from when this sentence was next to one referring to the Meta Environment. If they're loading this file, we can assume they just want the Docker process.

---

Added in https://github.com/WordPress/wordcamp.org/pull/113/commits/936ea89eb0aaaedcc90dda6925af03fdac01ad41